### PR TITLE
feat: Adding Light-Bridge GoldSky subgraphs

### PR DESCRIPTION
**Pull request by wsdt**
# Changes
- Added superior `boba` folder in `/packages` as we used to have in the legacy boba repo. If there is now a different place for our custom packages please let me know & I'll move it. 

- Added the subgraphs package, but only moved the new GoldSky subgraph for light bridge since it looks like we don't really use any other subgraphs (will check that in depth on the gateway and with the team). Depending on that I might deploy those subgraphs too and add the config files here. 
- Moved subgraphs for Dao 
- Added documentation.

### CODE REVIEW

File: packages/boba/subgraphs/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+Here is an example:
+`goldsky subgraph deploy light-bridge/v1 --from-abi ./lightbridge.json`
+
+## API / Account
+We already have a GoldSky account. As of 18. Dec 23 following people have already access that you can reach out to for access:
+- Ino
+- Souradeep
+- Kevin
+
```
Comment (souradeep-das) [inline packages/boba/subgraphs/README.md:28]: lets add Boyuan too!

**Review by souradeep-das — APPROVED** [review]
lgtm!

File: packages/boba/subgraphs/README.md (hunk 1 above)
Comment (wsdt) [reply packages/boba/subgraphs/README.md:28]: done!

**Comment by wsdt** [comment]
added you @Boyuan-Chen to the Goldsky account as well

**Comment by boyuan-chen** [comment]
> added you @Boyuan-Chen to the Goldsky account as well

Thank you. I activated my account.

File: packages/boba/subgraphs/local/README.md
Comment (sk-enya) [inline packages/boba/subgraphs/local/README.md:1]: Looks like auto generate local files by goldsky do we need to keep this in project?

Comment (wsdt) [reply packages/boba/subgraphs/local/README.md:1]: no it's actually for debugging locally as per discussion with the Goldsky team on Telegram :-)

**Review by sk-enya — APPROVED** [review]

**Pull request merged**